react.js

```diff
@@ -0,0 +1,71 @@
+module.exports = {
+	extends: ["airbnb", "airbnb/hooks"],
+	plugins: [],
```

Comment:
Don't you need to use plugins here?  Especially the `react` one. Or are they already included in `airbnb`?